node/src/command.rs

```diff
@@ -40,6 +40,8 @@ use std::{io::Write, net::SocketAddr, str::FromStr};
 fn load_spec(
 	id: &str,
 	para_id: ParaId,
+	mnemonic: &Option<String>,
+	accounts: u32,
```

Comment:
I don't think it makes sense to add these here considering they will be ignored for most chains (eg alphanet and mainnet in the future).